pom.xml

```diff
@@ -22,7 +22,7 @@
 
     <properties>
     	<openMRSVersion>2.0.5</openMRSVersion>
-    	<emrapiVersion>1.21.0</emrapiVersion>
+    	<emrapiVersion>1.24.0-SNAPSHOT</emrapiVersion>
```

Comment:
Is this the version you intended? 😊

Reply:
It is not, I forgot to squash my commits (oops!) One moment...